packages/driver/src/cy/net-stubbing/static-response-utils.ts

```diff
@@ -43,9 +43,17 @@ export function validateStaticResponse (cmd: string, staticResponse: StaticRespo
     err('`throttleKbps` must be a finite, positive number.')
   }
 
+  if (delayMs && delay) {
+    err('`delayMs` and `delay` cannot both be set.')
```

Comment:
Should this be more like "`delayMs` has been deprecated. Use `delay` instead.`"?

Reply:
Hmm, I tried thinking of a better error, but it's backwards-compatible so the wording is tricky. I think this wording is prob fine - we still support `delayMs`, so this is the most accurate error. In the future when `delayMs` is removed it can be updated.